### PR TITLE
ft(order): A buyer and seller  should be able to watch their orders

### PR DESCRIPTION
**Pull request by yvanddniyo**
**What does this PR do?**
This PR allows seller and buyer should be see their orders.

**Description of Task to be completed?**

a buyer should be able to see order has purchased
a seller should be able to see orders has been made on his or her products

**How should this be manually tested?**

Clone the repository.
Checkout to the branch ft-order-management
Run npm install to install dependencies.
Run npm run dev to start the development server and login
navigate to /orders
Login and get your orders 
Congratulation you have successfully seen your orders status

**What are the relevant pivotal tracker stories?**

Delivers (This task is not on the pivot tracker)

### CODE REVIEW

**Review by Heisjabo — APPROVED** [review]

**Review by Jovz19200 — APPROVED** [review]

File: src/controllers/paymentController.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const handleSuccess = async (req: Request, res: Response) => {
     try {
         const { userId, sessionId } = req.query as { userId: string, sessionId: string }
         const session = await stripe.checkout.sessions.retrieve(sessionId);
-        const user: any = await findUserById(userId);
+        const user: any = await findUserById(userId);``
```
Comment (teerenzo) [inline src/controllers/paymentController.ts]: why  this ``

File: src/services/user.service.ts [hunk 2]
```diff
@@ -262,6 +262,7 @@ export const  sendResetLinkEmail = async (email: any) =>{
       message: 'Password reset link sent to your email.',
 };
 } catch (error) {
+  console.log(error)
```
Comment (teerenzo) [inline src/services/user.service.ts]: remove this

**Review by teerenzo — CHANGES_REQUESTED** [review]

File: src/controllers/paymentController.ts (hunk 1 above)
Comment (yvanddniyo) [reply src/controllers/paymentController.ts]: Nothing I forgot to delete it.

File: src/services/user.service.ts (hunk 2 above)
Comment (yvanddniyo) [reply src/services/user.service.ts]: Yeah i removed it...

**Pull request merged**